libs/dmlf/include/dmlf/colearn/muddle_learner_networker_impl.hpp

```diff
@@ -40,36 +40,39 @@ namespace colearn {
 class MuddleLearnerNetworkerImpl : public AbstractMessageController
 {
 public:
-  using Taskpool                      = oef::base::Taskpool;
-  using Threadpool                    = oef::base::Threadpool;
-  using TaskP                         = Taskpool::TaskP;
+  using Taskpool = oef::base::Taskpool;
+
+  using Address                       = fetch::muddle::Address;
+  using AlgorithmClass                = AbstractMessageController::AlgorithmClass;
+  using Bytes                         = ColearnUpdate::Data;
+  using ConstUpdatePtr                = AbstractMessageController::ConstUpdatePtr;
+  using Criteria                      = UpdateStoreInterface::Criteria;
+  using Lock                          = std::unique_lock<Mutex>;
   using MuddlePtr                     = muddle::MuddlePtr;
-  using deprecated_UpdateInterfacePtr = dmlf::deprecated_UpdateInterfacePtr;
-  using RpcClient                     = fetch::muddle::rpc::Client;
-  using RpcClientPtr                  = std::shared_ptr<RpcClient>;
+  using NetMan                        = fetch::network::NetworkManager;
+  using NetManP                       = std::shared_ptr<NetMan>;
```

Comment:
the naming convention other people have used in the library seems to be NetManPtr for such cases

Reply:
there's seems to be no standard naming in this file though?
some thingP and some thingPtrs, would be nice to unify them

Reply:
An excellent point. Done.